src/favorites.html

```diff
@@ -38,6 +38,14 @@
       <load src="./partials/footer.html" />
     </footer>
 
+    <div class="scroll-up-button-container">
+      <button class="scroll-up-button scroll-up-button pulse">
+        <svg class="scroll-up-button-icon">
+          <use href="./images/icons.svg#icon-arrow"></use>
+        </svg>
+      </button>
+    </div>
+
```

Comment:
can it be moved to partial html file and loaded here and on index.html page?

Reply:
good point